<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonChoose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Metadane</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Podaj ścieżkę do metadanych</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zastosuj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
